geopandas/geodataframe.py

```diff
@@ -122,15 +122,17 @@ def set_geometry(self, col, drop=False, inplace=False, crs=None):
 
         Parameters
         ----------
-        col : column label or array
-        drop : boolean, default True
+        col: column label or array
+        drop: boolean, default True
```

Comment:
Please leave those spaces, that's for numpydoc

Reply:
Strange that it doesn't work with Sphinx. I wonder if there is a setting somewhere that needs to be tweaked ...

Reply:
It's restructuredtext .. (see formatting of definition list: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists), and how numpydoc parses the content (there has been discussion to make it less strict: https://github.com/numpy/numpydoc/issues/78, but currently numpydoc requires a space before the colon)